schemaRegistry/userEvents-v1/userEvent.ts

```diff
@@ -0,0 +1,31 @@
+/***
+ * User-Event schema class generated by the schema registry code bindings
+ */
+export class Event {
```

Comment:
this is the class generated by the schema registry code bindings.
in long term, this folder will be packaged as a npm library so that the event class can be used by both publisher and subscriber

question to the team:
- do we need the schema registry for this purpose (or) create our own library that has the simplified types like this?: 
```
type UserEvent = {
  email: string,
  userId: string,
  isPremium: boolean
}
```

Reply:
I think `status` is included in this event, fyi. Not sure exactly what it means though. @kkyeboah ?